os/include/tinyara/binary_manager.h

```diff
@@ -67,12 +67,12 @@
 /* The lenght of user or kernel partition path */
 #define BINARY_PATH_LEN                  16
 
-/* Binary Type */
-enum binary_type_e {
-	BINARY_KERNEL = 1,
-	BINARY_COMMON = 2,
-	BINARY_USERAPP = 3,
-	BINARY_TYPE_MAX,
+/* Binary Group Type : Kernel, Common, User app */
+enum binary_group_type_e {
+	BINARY_KERNEL = 0,
+	BINARY_COMMON = 1,
+	BINARY_USERAPP = 2,
+	BINARY_TYPE_MAX = 3,
```

Comment:
1. I adjust the values to user these for return type of binary_manager_set_bootparam, `binary_setbp_result_t`.
This function requires three result values because of the number of type is 3 (kernel, common, userapp).
So I adjust these values.

2. 'binmgr_result_type_e binary_manager_set_bootparam(uint8_t type, binary_setbp_result_t *update_result)`
`type` is parameter value user set the binaries that are need to update together.
`binary_group_type_e` is used for setting of binary type with `BM_XXXX_GROUP`.